tools/data/sthv1/preparing_sthv1.md

```diff
@@ -7,16 +7,32 @@ Before we start, please make sure that the directory is located at `$MMACTION2/t
 
 First of all, you have to sign in and download annotations to `$MMACTION2/data/sthv1/annotations` on the official [website](https://20bn.com/datasets/something-something/v1).
 
-## Step 2. Prepare Videos
+## Step 2. Prepare RGB Frames / Videos
 
-Then, you can download all data parts to `$MMACTION2/data/sthv1/` and use the following command to extract.
+In Step 2, you have two choices: directly downloading RGB frames on [sthv1 website](https://20bn.com/datasets/something-something/v1) or downloading videos on [sthv2 website](https://20bn.com/datasets/something-something/v2) and extracting frames.
```

Comment:
Since the [sthv1 website](https://20bn.com/datasets/something-something/v1) doesn't provide the original video data and only extracted RGB frames are available, here are two alternatives to choose: 
a.  For users who only want to use RGB frames, you can directly download RGB frames from [sthv1 website](https://20bn.com/datasets/something-something/v1). 
b. For users who need videos and RGB + Flow, you should download the videos from [sthv2 website](https://20bn.com/datasets/something-something/v2) and go with the following steps to extract them.